### DISCUSSION TITLE
Lost focus upon saving an edit of an annotation

### DISCUSSION THREAD

**Discussion by rhoadsstevens** [ManifoldScholar/manifold#3478] [Accessibility]
Priority level 2 (out of 3) from UW Accessible Technologies

WCAG mapping: 2.4.7 Focus Visible

STEPS TO REPRODUCE:
1. Select a book (i.e. ""I My Recall"")
2. Turn on caret browsing (insert + F7)
3. Tab into the text
4. Arrow through the text to a comment. Press enter to open the comment
5. Tab to the save button and press save. Look for the focus indicator.

OBSERVED BEHAVIOR
When you tab to the save button and press save, the fous indicator disappears. This causes confusion and requires the user to press tab additionally to find the focus indicator (in this case, the focus will then go to the "reply" button after pressing tab).

EXPECTED BEHAVIOR:
Focus indicator should land where you last left off. In this case, where you left off was pressing "save" for the "edit" function. Since the process started on the "edit" button, the focus indicator should resume the overall process by landing back on the "edit" button.

**Comment by dananjohnson** [comment]
This was resolved in v9.
